sdks/python/apache_beam/coders/coder_impl.pxd

```diff
@@ -137,3 +138,6 @@ cdef class WindowedValueCoderImpl(StreamCoderImpl):
 
   @cython.locals(c=CoderImpl)
   cpdef get_estimated_size_and_observables(self, value, bint nested=?)
+
+  @cython.locals(wv=windowed_value.WindowedValue)
+  cpdef encode_to_stream(self, value, OutputStream stream, bint nested)
```

Comment:
This code is only to declare the type of wv (which we're doing here in the pxd file to keep Cython optional).